### DISCUSSION TITLE
yaegi test issue on request.go [request.go:605:48: cannot use type resty.RetryConditionFunc as type func(*unsafe2.dummy, error) bool]

### DISCUSSION THREAD

**Discussion by gordonlukch** [go-resty/resty#608] [General]
Hi all,

I am trying to create a Traefik Proxy plugin, but i got a issue on the yaegi test, it point to request.go as below:
github.com/go-resty/resty/v2/request.go:605:48: cannot use type resty.RetryConditionFunc as type func(*unsafe2.dummy, error) bool

any idea for the issue? how can i overcome this. Thank you very much.

The bot analyzer full report:
https://github.com/gordonlukch/Keycloak-Introspect/issues/7

**Comment by moorereason** [comment]
I'm not sure where the `func (*unsafe2.dummy, error) bool` type is coming from.  I doubt this is a problem inside of resty.
